src/pytorch_lightning/plugins/precision/colossalai.py

```diff
@@ -13,7 +14,15 @@
 
 
 class ColossalAIPrecisionPlugin(PrecisionPlugin):
-    def __init__(self) -> None:
+    def __init__(self, precision: Union[str, int] = 16, amp_level: Optional[str] = None) -> None:
```

Comment:
we can remove amp_level since we are going to deprecate it soon, so let's not add it.

Reply:
Done.